locale/zh/messages.json

```diff
@@ -247,8 +247,6 @@
   "Disk (MiB)": "磁盘 (Mib)",
   "Disk Allocation": "磁盘分配",
   "Disk Consumed": "",
-  "Disk Space (MiB)": "磁盘空间 (MiB)",
```

Comment:
🤔 we should ensure we want to lose these

Reply:
I checked, they no longer appear.